src/frontend/src/lib/utils/token.utils.ts

```diff
@@ -190,7 +190,19 @@ const createTokenGroup = ({
 	},
 	nativeToken,
 	nativeNetwork: nativeToken.network,
-	tokens: [nativeToken, twinToken]
+	tokens: [nativeToken, twinToken],
+	balance:
+		nonNullish(nativeToken.balance) && nonNullish(twinToken.balance)
```

Comment:
- should we extract a small inner function for readability?

- can you add a comment that we assume that twin token and native token are always 1 to 1? e.g. 1 ckBTC === 1 BTC

- should we extend the test?

Reply:
- done! and created test

- good catch! I completely missed that... 

- see first point

please review it again, because i separated the smaller functions and created tests about it